library/worldgen/surface_rule/src/main/java/org/quiltmc/qsl/worldgen/surface_rule/impl/QuiltSurfaceRuleInitializer.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2022 QuiltMC
+ * Copyright 2022-2023 QuiltMC
```

Comment:
```suggestion
 * Copyright 2022 QuiltMC
```

After some talks, headers should not be updated anymore for existing files.

Reply:
Does running applyLicenses now correctly apply licenses? If so, I'll just run that and see the results

Reply:
I don't think it will do it accurately yet.

Reply:
Should be fixed by latest commit